server/athenian/api/internal/features/entries.py

```diff
@@ -1340,15 +1335,14 @@ async def calc_jira_histograms(
     ) -> np.ndarray:
         """Calculate histograms over JIRA issues."""
         reporters, assignees, commenters = self._merge_jira_participants(participants)
+        jira_filter = JIRAFilter.from_jira_config(jira_ids).replace(
```

Comment:
It's better to rename this to `jira_filters` because we've typically got more than one.